src/index.js

```diff
@@ -1,37 +1,23 @@
 import './style.css';
+import getFromLocalSTorage from './myLocalStorage.js';
+import displayTodo from './displayTodo.js';
 
-const taskContent = document.querySelector('.todo-header');
+window.addEventListener('load', () => {
+  const todos = getFromLocalSTorage();
+  const newTodoForm = document.querySelector('#new-todo-form');
 
-const tasks = [
-  {
-    description: 'Read JavaScript',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Read CSS',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Work on HTML',
-    completed: false,
-    index: 3,
-  },
-];
+  newTodoForm.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const todo = {
+      content: e.target.elements.content.value,
+      done: false,
+    };
 
-const displayTask = (tasks) => {
-  tasks.forEach((task) => {
-    const html = `
-    <div class="header">
-    <div class="check-content">
-      <input type="checkbox" class="checkbox" />
-      <div class="task"><p>${task.description}</p></div>
-    </div>
-    <i class="fa-solid fa-ellipsis-vertical"></i>
-  </div>`;
-    taskContent.insertAdjacentHTML('beforeend', html);
-  });
-};
+    todos.push(todo);
+    localStorage.setItem('todos', JSON.stringify(todos));
+    e.target.reset();
```

Comment:
- [ ] Great job on the functionality for adding and deleting but it shouldn't presist in the page so after deleting the tasks and creating a new task I can see them again in the page so please delete it and don't let the task presist in the page 😊🙏🏻🔥✨📌

Reply:
Done. Thanks